docs/sources/get-started/deployment-modes.md

```diff
@@ -8,101 +8,81 @@ aliases:
+-- Distributor
+-- Ingester
+- `-target=read` - The read target is stateless and can be run as a Kubernetes Deployment that can be scaled automatically. It contains the following components:
+-- Query front end
+-- Queriers
+- `-target=backend` - The backend target is stateful, and is controlled by a Kubernetes StatefulSet. Contains the following components:
+-- Compactor
+-- Index gateways
+-- Query scheduler
+-- Ruler
 
-## Simple scalable deployment mode
+The scalable monolithic deployment mode requires a reverse proxy to be deployed in front of Loki, to direct client API requests to either the read or write nodes. The Loki Helm chart includes a default reverse proxy configuration, using Nginx.
 
-If your volume of logs exceeds a few hundred GB a day,
-or if you would like to separate read and write concerns,
-Loki provides the simple scalable deployment mode.
-This deployment mode can scale to several TBs of logs per day and more.
-Consider the microservices mode approach for very large Loki installations.
+## Monolithic mode
 
-![Simple scalable deployment mode diagram](../simple-scalable.png "Simple scalable deployment mode")
+The simplest mode of operation is monolithic mode. You enable monolithic mode by setting the `-target=all` command line parameter. This mode runs all of Loki’s microservice components inside a single process as a single binary or Docker image.
 
-In this mode the component microservices of Loki are bundled into two targets:
-`-target=read` and `-target=write`.
-The BoltDB [compactor]({{< relref "../operations/storage/boltdb-shipper#compactor" >}}) 
-service will run as part of the read target.
+![monolithic mode diagram](../monolithic-mode.png "Monolithic mode")
 
-There are advantages to separating the read and write paths:
+Monolithic mode is useful for getting started quickly to experiment with Loki, as well as for small read/write volumes of up to approximately 100GB per day.
 
-* higher availability of write path by providing dedicated nodes
-* separately scalable read path to add/remove query performance on demand
+You can horizontally scale a monolithic mode deployment to more instances by using a shared object store, and by configuring the [`ring` section]({{< relref "../configure#common" >}}) of the `loki.yaml` file to share state between all instances, but the recommendation is to use scalable monolithic mode if you need to scale your deployment.
 
-The simple scalable deployment mode requires a load balancer in front of Loki,
-which directs `/loki/api/v1/push` traffic to the write nodes.
-All other requests go to the read nodes.
-Traffic should be sent in a round robin fashion.
+You can configure high availability by running two Loki instances using `memberlist_config` configuration and a shared object store. You route traffic to all the Loki instances in a round robin fashion.
```

Comment:
note: you also need to enable replication_factor: 3 to get HA with Loki (this is true for SSD as well). Running multiple instances with rep factor of 1 will not give you HA.